actions/setup-conftest/action.yaml

```
@@ -18,6 +18,14 @@ runs:
         path: ${{ github.workspace }}/bin/conftest
         key: conftest-${{ runner.os }}-${{ runner.arch }}-${{ inputs.version }}
 
+    - name: Normalize runner.os to match Conftest release
+      shell: sh
+      env:
+        OS: ${{ runner.os }}
+      run: |
+        [[ "$OS" == 'macOs' ]] && OS='Darwin'
+        echo "OS=$OS" >> "$GITHUB_ENV"
```

can't we do it the same way we do the two below?

```suggestion
      shell: sh
      if: runner.os == 'macOs'
      run: |
        echo "OS=Darwin" >> "$GITHUB_ENV"
```